packages/manager/modules/pci/src/projects/project/private-networks/private-networks.controller.js

```diff
@@ -5,15 +5,89 @@ const CONTAINER_NAME = 'pci.projects.project.privateNetwork';
 
 export default class {
   /* @ngInject */
-  constructor($translate, CucCloudMessage) {
+  constructor($translate, $q, CucCloudMessage, PciPrivateNetworks) {
     this.$translate = $translate;
+    this.$q = $q;
     this.CucCloudMessage = CucCloudMessage;
     this.PRIVATE_NETWORK_LIST = PRIVATE_NETWORK_LIST;
+    this.PciPrivateNetworks = PciPrivateNetworks;
   }
 
   $onInit() {
     this.loadMessages();
+    this.isLoading = false;
     this.criteria = getCriteria('id', this.networkId);
+    this.availableNetworks = null;
+    this.gatewaySubnetObj = null;
+    this.networksSubnet();
```

Comment:
**nitpick:** Please prefix the call with verb.
```suggestion
    this.getNetworksSubnet();
```

Reply:
Taken care.